install-sh

```diff
@@ -232,7 +232,7 @@ if test -z "$dir_arg"; then
       else
         u_plus_rw='% 200'
       fi
-      cp_umask=`expr '(' 777 - $mode % 1000 ')' $u_plus_rw`;;
+      cp_umask=$(expr '(' 777 - $mode % 1000 ')' $u_plus_rw);;
```

Comment:
`install-sh` is a standard file that is used in many projects, so why edit it.

Reply:
Sorry, didn't know that as it wasn't in the .gitignore. I can remove that commit if you want me too, or update the original